src/Evolution/Executables/GeneralizedHarmonic/EvolveGeneralizedHarmonicSingleBlackHole.hpp

```diff
@@ -93,21 +93,20 @@ struct EvolutionMetavars
   struct ExcisionBoundaryA
       : tt::ConformsTo<intrp::protocols::InterpolationTargetTag> {
     using temporal_id = ::Tags::Time;
-    using tags_to_observe = tmpl::list<
-        gr::Tags::LapseCompute<volume_dim, ::Frame::Inertial, DataVector>>;
+    using tags_to_observe = tmpl::list<gr::Tags::Lapse<DataVector>>;
     using compute_vars_to_interpolate =
         ah::ComputeExcisionBoundaryVolumeQuantities;
     using vars_to_interpolate_to_target =
-        tmpl::list<gr::Tags::SpacetimeMetric<volume_dim, Frame::Inertial>>;
+        tmpl::list<gr::Tags::Lapse<DataVector>, gr::Tags::Shift<3, Frame::Grid>,
+                   gr::Tags::SpatialMetric<3, Frame::Grid>>;
     using compute_items_on_source = tmpl::list<>;
     using compute_items_on_target = tmpl::append<tmpl::list<
-        gr::Tags::SpatialMetricCompute<volume_dim, Frame::Inertial, DataVector>,
-        gr::Tags::DetAndInverseSpatialMetricCompute<volume_dim, Frame::Inertial,
-                                                    DataVector>,
-        gr::Tags::ShiftCompute<volume_dim, Frame::Inertial, DataVector>,
-        gr::Tags::LapseCompute<volume_dim, Frame::Inertial, DataVector>,
-        GeneralizedHarmonic::ConstraintDamping::Tags::ConstraintGamma1>>;
-
+        GeneralizedHarmonic::ConstraintDamping::Tags::ConstraintGamma1,
```

Comment:
We need:
1. a compute tag to get InverseSpatialMetric from the grid SpatialMetric
2. Use `ah::Tags::OneOverOneFormMagnitudeCompute<3, Frame::Grid>`
3. Use `ah::Tags::UnitNormalOneFormCompute<Frame::Grid>`
4. A new `CharacteristicCompute` tag that uses `ah::Tags::UnitNormalOneForm<Frame::Grid>` as the normal covector